src/transformStateWithClones.js

```diff
@@ -7,7 +7,58 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  let newObj;
+  let count = 0;
```

Comment:
Then this count, why do you need it, you can do a copy of a previous value of 'state', on each iteration form an array that you've declared above